docs/developers/modules/marketmap.mdx

```diff
@@ -0,0 +1,292 @@
+---
+description: Reference documentation for the Marketmap module.
+title: Marketmap
+icon: message-dollar
+---
+
+## Overview
+
+The `Marketmap` module is responsible for managing and storing a configuration that informs the Connect oracle of which markets to fetch data for, and which providers to use to fetch them.
+
+The `Marketmap` module's Keeper is instantiated with an `Authority` address, guarding the mutation of the `Marketmap` by a specific address. This address can be the `governance` address, allowing governance propositions to mutate the `Marketmap`.
+
+The `Authority` address can also update the Keeper's `MarketAuthorities`, allowing for additional addresses to update the `Marketmap`.
```

Comment:
I think we could draw a stronger distinction between these two types of addresses.  The `Admin` address is able to change the `MarketAuthorities`.

Reply:
[reword](https://github.com/skip-mev/slinky/pull/630/commits/4e675d77132c2501a30b75f85ad196720c7f7bd0)

i made some adjustments, but not entirely sure if this addresses your comment. let me know if you had something more in mind